README.md

```diff
@@ -252,6 +255,22 @@ Run the tests with the following command:
 docker build . -t s3-resource --build-arg S3_TESTING_ACCESS_KEY_ID="access-key" --build-arg S3_TESTING_SECRET_ACCESS_KEY="some-secret" --build-arg S3_TESTING_BUCKET="bucket-non-versioned" --build-arg S3_VERSIONED_TESTING_BUCKET="bucket-versioned" --build-arg S3_TESTING_REGION="us-east-1" --build-arg S3_ENDPOINT="https://s3.amazonaws.com"
 ```
 
+This resource also supports assuming a role. Here's how you test that:
+
+1. Do not permit the test user you are using to access the S3 bucket, e.g.
+give the user a policy that only permits access to the two buckets (versioned
+and unversioned).
```

Comment:
I found this super confusing. I'd recommend stating explicitly which permissions the test user should and should not have for each bucket, so that's it's clear and unambiguous. My best guess is that you're saying the test user should have permissions for S3_TESTING_BUCKET and S3_VERSIONED_TESTING_BUCKET but not S3_ROLE_RESTRICTED_TESTING_BUCKET - is that right?

Reply:
Yep that's right. Sorry it was not clear. I've rewritten this - please let me know what you think.

Reply:
Thanks! I can follow it easily now :)